src/main/scala/ai/eto/rikai/sql/model/Catalog.scala

```diff
@@ -75,6 +75,7 @@ object Catalog {
           .asInstanceOf[Catalog]
       )
     }
+    println(s"catalog get ${catalog.get.getClass}")
```

Comment:
remove println or change it to `logger.debug`?

Reply:
updated